audio_service_platform_interface/lib/method_channel_audio_service.dart

```diff
@@ -78,162 +80,167 @@ class MethodChannelAudioService extends AudioServicePlatform {
     });
   }
 
+  @override
   void setHandlerCallbacks(AudioHandlerCallbacks callbacks) {
     _handlerChannel.setMethodCallHandler((call) async {
       switch (call.method) {
         case 'getChildren':
           return (await callbacks.getChildren(GetChildrenRequest(
-                  parentMediaId: call.arguments['parentMediaId'],
-                  options: call.arguments['options'])))
```

Comment:
this is currently in the `one-isolate`
it will fail casting, because `options` is `Map<String, dynamic>?` and `call.arguments['options']` is `Map<dynamic, dynamic>`

Reply:
Scratch my previous comment - this `_castMap` isn't going away with `invokeMap`. But otherwise, good find. However, note the map also needs to be nullable. Probably the example always passes a null here which is why the error didn't show up. But this will introduce new errors if it's not nullable.

Reply:
yeah, fixed, nice catch